dhall-docs/src/Dhall/Docs.hs

```diff
@@ -74,10 +50,10 @@ parseOptions =
        <> Options.Applicative.metavar "INPUT"
        <> Options.Applicative.help "Directory of your dhall package" )
     <*> Options.Applicative.strOption
-        ( Options.Applicative.long "output"
-       <> Options.Applicative.metavar "OUTPUT"
-       <> Options.Applicative.help "Directory where your docs will be generated"
-       <> Options.Applicative.value "docs" )
+        ( Options.Applicative.long "output-link"
+       <> Options.Applicative.metavar "OUTPUT-LINK"
+       <> Options.Applicative.help "Path for the link to the generated documentation"
+       <> Options.Applicative.value "./docs/index.html" )
```

Comment:
I would create a symlink to the directory rather than a symlink to the `index.html` file.  That makes it easier for the user to access generated files other than `./docs/index.html` from the command line.  This is also the way Nix tends to work